### PR TITLE
Update runtime to GNOME 46

### PR DESCRIPTION
**Pull request by hfiguiere**

### CODE REVIEW

File: org.gnome.Photos.json [hunk 1]
```diff
@@ -294,13 +294,14 @@
                 "-Dminer_fs=true",
                 "-Dminer_fs_cache_location=$XDG_CACHE_HOME/gnome-photos/miner/files",
                 "-Dminer_rss=false",
+		"-Dbattery_detection=none",
```
Comment (A6GibKm) [inline org.gnome.Photos.json]: There is a new option here https://gitlab.gnome.org/GNOME/tracker-miners/-/blob/master/meson_options.txt?ref_type=heads#L22. I will ask what it does.

File: patches/gnome-photos-appadata.patch [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+   </description>
++  <launchable type="desktop-id">org.gnome.Photos.desktop</launchable>
+   <kudos>
+     <kudo>AppMenu</kudo>
+     <kudo>HiDpiIcon</kudo>
+@@ -104,7 +105,6 @@
+     </screenshot>
+   </screenshots>
+   <url type="bugtracker">https://gitlab.gnome.org/GNOME/gnome-photos/issues</url>
+-  <url type="donation">http://www.gnome.org/friends</url>
```
Comment (A6GibKm) [inline patches/gnome-photos-appadata.patch:26]: https://www.gnome.org/donate/

File: org.gnome.Photos.json (hunk 1 above)
Comment (hfiguiere) [reply org.gnome.Photos.json]: I added this because otherwise the build fail.

Comment (hfiguiere) [reply org.gnome.Photos.json]: fixed the indentation however. Emacs started adding tabs in JSON and it drives me nuts.

File: patches/gnome-photos-appadata.patch (hunk 2 above)
Comment (hfiguiere) [reply patches/gnome-photos-appadata.patch:26]: upstream can fix it.
This fail validation.

File: org.gnome.Photos.json [hunk 3]
```diff
@@ -294,13 +294,14 @@
...
                 "-Dminer_rss=false",
+                "-Dbattery_detection=none",
                 "-Dsystemd_user_services=false"
             ],
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/tracker-miners/3.4/tracker-miners-3.4.3.tar.xz",
-                    "sha256": "8e4f3976472641b648d0f8f20debae1b1629ca5b560b8616e117d29d756fbeeb",
+                    "url": "https://download.gnome.org/sources/tracker-miners/3.7/tracker-miners-3.7.0.tar.xz",
```
Comment (hfiguiere) [inline org.gnome.Photos.json:303]: Note: the gnome f-e-d-c want to downgrade because odd version number.

Comment (A6GibKm) [reply org.gnome.Photos.json:303]: stable-only: false. I should go back to the checker and have a test for tracker-miners

Comment (A6GibKm) [reply org.gnome.Photos.json:303]: Or just stop pretending there are modules releasing with the old 3.XYZ versioning scheme.

Comment (hfiguiere) [reply org.gnome.Photos.json:303]: Added `stable-only: false` and f-e-d-c no longer complains.

**Comment by hfiguiere** [comment]
works here.

**Pull request merged**